### ISSUE TITLE
Get javascript error with TinyMce Editor by calling it if plugin is activated

### ISSUE THREAD

**Issue by Portugao** [Guite/MostGenerator#666]
```
SyntaxError: expected expression, got '<' plugin.min.js:1:0
```

**Comment by Guite** [comment]
Please describe how you activated the plugin step by step.

**Comment by Portugao** [comment]
I put the relevant files for TinyMce into the plugin folder of Scribite.
Then i activated the plugin of MUVideo in the settings of TinyMce.

**Comment by Guite** [comment]
Fixed in cefeedea2fa8c3499e3544cb9722d0f458420838

**Comment by Portugao** [comment]
:+1:
